frontend/src/index.css

```
@@ -356,6 +356,76 @@
+  --secondary: 240 5% 96%;
+  --secondary-foreground: 240 7% 18%;
+  --muted: 240 5% 96%;
+  --muted-foreground: 240 4% 42%;
+  --accent: 240 5% 94.5%;
+  --accent-foreground: 240 7% 16%;
+  --border: 240 5% 90%;
+  --input: 240 5% 86%;
+  --sidebar-background: 240 5% 97%;
+  --sidebar-foreground: 240 5% 30%;
+  --sidebar-border: 240 5% 90%;
+  background: hsl(var(--background));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)